recipes/byobu/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "byobu" %}
+{% set version = "5.127" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://launchpad.net/byobu/trunk/{{ version }}/+download/byobu_{{ version }}.orig.tar.gz
+  sha256: 4bafc7cb69ff5b0ab6998816d58cd1ef7175e5de75abc1dd7ffd6d5288a4f63b
+  patches:
+    # Ensure we use python in the same conda environment
+    - 0001-use-local-python.diff
+    # Ensure byobu conda environment is activated
+    - 0002-byobu-conda.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - make
+    - coreutils  # [osx]
+    - gettext    # [osx]
+  run:
+    - newt
+    - tmux >=1.5
+    # GNU Tools
+    - coreutils  # [osx]
```

Comment:
Why is this only a requirement of OSX? Given a minimal python/debian Docker container we might want to install those as well.

Reply:
In OSX byobu does not work at all with the bundled BSD utils, e.g. readlink is not functional.

I think it would be better to leave that as optional on Linux, i.e. if you are building a minimal python/Debian container it is probably better to just add `coreutils` to the list of packages in the environment.

What do you think?

Reply:
Your call :-) its not a common use case so feel free to merge!

Thanks!